app/_assets/images/hello.sh

```diff
@@ -0,0 +1 @@
+hello.s
```

Comment:
@subnetmarco Was this file left in here by accident?

Reply:
Yes please remove